Gaijinizer patch/Map235.txt

```diff
@@ -2,59 +2,60 @@
 > BEGIN STRING
 どう……？妲己。
 > CONTEXT: Map235/events/1/pages/0/10/Dialogue/kazuya
-How……? Daji.#MTLed
+What do you think, Daji?#Ron
 > END STRING
 
 > BEGIN STRING
 ふん！どうもなにも……
 
 一言で言うなら『脈ナシ』じゃ。
 > CONTEXT: Map235/events/1/pages/0/14/Dialogue/enemy17
-Hmm! Thank you very much...In a nutshell, it's
-a "pulseless pear."#MTLed
+Hmm! To put it in a nutshell, there's nothing I
+can do.#RonIffy
 > END STRING
 
 > BEGIN STRING
 あれだけ念入りにかけておいた変化の術式も切れておる
 
 この姿に完全に固定されている状態じゃ。
 > CONTEXT: Map235/events/1/pages/0/18/Dialogue/enemy17
-The technique of the change that I was careful
-too much is broken. It's completely fixed in this
-form.#MTLed
+The care and attention to deal put into this
+transformation magic has made this current form
+very stable...#RonIffy
 > END STRING
 
 > BEGIN STRING
 ヘカーテのやつめ……ただの改変創造ではないな。
 
 形式そのものから創り変えておるのか……
 > CONTEXT: Map235/events/1/pages/0/22/Dialogue/enemy17
-Hecate's cousin...It's not just a creative
-creation. Is it changing from the form itself
-......#MTLed
+That Hecate didn't just simply remold her into
+this form, she recreated the entire form itself.#Ron
 > END STRING
```

Comment:
a minor rewording that I think is clearer:
>_**That Hecate didn't just simply remold her into
this form, she remade her entire form from
scratch.**_